tests/third_party/cupy/core_tests/test_ndarray_unary_op.py

```diff
@@ -141,6 +141,7 @@ def test_invert_zerodim(self):
     *testing.product({"xp": [numpy, cupy], "shape": [(3, 2), (), (3, 0, 2)]})
 )
 class TestBoolNeg(unittest.TestCase):
+    @pytest.mark.skip("dpnp does not raise TypeError")
```

Comment:
Does new framework allow to easily handle a behavior different from dpctl for some use case?
Can we get rid of the limitation here and in substract call below for boolean type?

Reply:
Fix for this tests in dpctl.tensor functions